stripe/src/main/java/com/stripe/android/exception/StripeException.java

```diff
@@ -3,44 +3,54 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
+import com.stripe.android.StripeError;
+
 /**
  * A base class for Stripe-related {@link Exception Exceptions}.
  */
 public abstract class StripeException extends Exception {
 
     protected static final long serialVersionUID = 1L;
 
-    @Nullable private final String requestId;
-    @Nullable private final Integer statusCode;
+    @Nullable private final String mRequestId;
+    @Nullable private final Integer mStatusCode;
+    @Nullable private final StripeError mStripeError;
 
     public StripeException(@Nullable String message, @Nullable String requestId,
```

Comment:
since this is public, does this change count as a breaking change? e.g. if someone is creating their own StripeException for some reason this change in method signature would break that code?

Reply:
Good point. I think the constructor was likely incorrectly marked as `public` as there isn't a reason I can think of why someone would instantiate their own `StripeException`. That said, I can preserve the existing constructors so that this isn't a breaking change.

Reply:
wfm